test/lit.site.cfg.in

```diff
@@ -24,6 +24,7 @@ config.llvm_use_sanitizer = "@LLVM_USE_SANITIZER@"
 config.have_zlib = "@HAVE_LIBZ@"
 config.enable_shared = @ENABLE_SHARED@
 config.enable_backtrace = "@ENABLE_BACKTRACES@"
+config.enable_quadfp = @TARGET_SUPPORTS_QUADFP@
```

Comment:
please enclose @TARGET_SUPPORTS_QUADFP@  within quotes similar to other assignments.

Reply:
`@TARGET_SUPPORTS_QUADFP@` expands to a Python boolean value (either `True` or `False`). If you quote the value and turn it into a string, then the expression will also evaluate to `True`.

Reply:
@shivaramaarao Is it okay to leave the code as-is?

Reply:
Yes. it is fine with me. Thanks